test/sanity/code-smell/botmeta.py

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env python
+"""Make the data in BOTMETA.yml is valid"""
+
+import sys
+import yaml
+
+from voluptuous import Any, Required, Schema
+from voluptuous.humanize import humanize_error
+
+from ansible.module_utils.six import string_types
+list_string_types = list(string_types)
+
+
+def main():
+    for path in sys.argv[1:] or sys.stdin.read().splitlines():
+        with open(path, 'r') as f:
+            botmeta = yaml.safe_load(f)
+
+        files_schema = Any(
+            Schema(*string_types),
+            Schema({
+                'ignored': Any(list_string_types, *string_types),
+                'keywords': Any(list_string_types, *string_types),
+                'labels': Any(list_string_types, *string_types),
+                'maintainers': Any(list_string_types, *string_types),
+                'notified': Any(list_string_types, *string_types),
+                'support': Any("core", "network", "community"),
+            })
+        )
+
+        list_dict_file_schema = [{str_type: files_schema}
+                                 for str_type in string_types]
+
+        schema = Schema({
+            Required('automerge'): bool,
+            Required('files'): Any(None, *list_dict_file_schema),
+            Required('macros'): dict,  # Any(*list_macros_schema),
+        })
+
+        # We don't check that all the `$team_` names are valid
+        # or that GitHub account names exist
+
+        try:
+            schema(botmeta)
+        except Exception as e:
```

Comment:
`voluptuous.MultipleInvalid` should be used instead of `Exception` ([see](https://github.com/alecthomas/voluptuous/blob/e72fd3bfb24f89888f51825313de98c397641674/voluptuous/schema_builder.py#L268-L271) and [voluptuous.error](https://github.com/alecthomas/voluptuous/blob/master/voluptuous/error.py)).